docker-compose.yml

```diff
@@ -29,4 +29,6 @@ services:
       retries: 3
     restart: unless-stopped
     volumes:
+    #Replace with folders on your filesystem if you do not want the data held in the container 
```

Comment:
Having a comment on `/config` is ok, because a custom config folder is optional, but the comment is not really ok, as it's not about "data held" but more about "having a custom config"